cpp/dolfinx/fem/DofMap.cpp

```diff
@@ -93,8 +93,12 @@ fem::DofMap build_collapsed_dofmap(MPI_Comm comm, const DofMap& dofmap_view,
 
   // Send new global indices for owned dofs to non-owning process, and
   // receive new global indices from owner
-  std::vector global_index_remote
-      = dofmap_view.index_map->scatter_fwd(global_index, 1);
+
+  std::vector<std::int64_t> global_index_remote(
+      dofmap_view.index_map->num_ghosts());
+  dofmap_view.index_map->scatter_fwd(
+      tcb::span<const std::int64_t>(global_index),
```

Comment:
I think this could be just `global_index`.

Reply:
gcc won't compile that.